src/cli/options.rs

```diff
@@ -2352,21 +2344,21 @@ pub enum CaCommand {
     Delete(Handle),     // Delete the CA -> let it withdraw and request revocation as well
 
     // Publishing
-    RepoPublisherRequest(Handle), // Get the RFC8183 publisher request
+    RepoPublisherRequest(Handle), // Get the idexchange:: publisher request
     RepoDetails(Handle),
     RepoUpdate(Handle, RepositoryContact),
     RepoStatus(Handle),
 
     // Parents (to this CA)
-    ChildRequest(Handle), // Get the RFC8183 child request
+    ChildRequest(Handle), // Get the idexchange:: child request
```

Comment:
You probably didn't mean to change this ;-)

Reply:
nope, fixed